### DISCUSSION TITLE
caching 0%

### DISCUSSION THREAD

**Discussion by djmips** [friction2d/friction#102] [Q&A]
I don't know if this is a bug or something else because I don't know exactly how I get in this state but sometimes I just see 'caching 0%' where the frame number would be in the animation play controls. I don't know how to get out of this currently so I just close and re-open? Any tips on how I can address this without restarting? Or is it a bug and I should try and figure out how to add it as an issue.

**Comment by rodlie** [comment]
![Untitled01](https://github.com/friction2d/friction/assets/34516798/e276504b-df18-42ae-843a-05187fdd4ad1)

Did you press button 2 (space)? If you have never played the project before you will need to press button 1 (shift+space). This is probably confusing and should be resolved. I will try to make this more logical.

> I don't know how to get out of this currently

Press Esc key (or press the stop button), then press button 1 (shift+space)?

**Reply by sherkhan30452** [reply]
Same with me, I rendered the final video, it was fine, but 0% cache in render preview.

Edit: I used raster (custom shaders)

**Reply by rodlie** [reply]
Need more info. Also check the console window for clues.

**Reply by sherkhan30452** [reply]
![image](https://github.com/friction2d/friction/assets/33783004/7289aa87-9740-4b6f-9d26-f9fb236ba0a1)

Will this help?

**Reply by sherkhan30452** [reply]
Windows 10 x64 Build 19045.4046, Intel(R) Core(TM) i3-8145U CPU @ 2.10GHz   2.30 GHz, RAM 16.0 GB (15.9 GB usable)
Friction version: friction-0.9.5-portable-win64
Edit: No GPU, only relying on Intel CPU

Case 1: I used Raster Effect from the [enve-plugins](https://github.com/axiomgraph/enve-plugins) by [axiomgraph](https://github.com/axiomgraph) using version 2.0 (it said it can be used for friction2d)

Case 2: Even without this Raster effect, a new file, with simple animation, for example moving circle from left to right, caching runs but it does not caches the whole project always (need to run button 1 multiple times) . See GIF. 
![friction frame issue 1 demo](https://github.com/friction2d/friction/assets/33783004/b762360f-67a1-4aa2-954e-556750b4bea0)

The console is not changing as shown in previous reply when performing any action.

**Reply by rodlie** [reply]
It's not the GPU, I have the same (UHD 620) on two computers I use for Friction. Are you running out of RAM?

**Reply by sherkhan30452** [reply]
No, but after restarting my pc, friction2d is working properly now. _(Also due to Windows update I had to restart my pc)_

Sorry for any inconvenience and late reply.
